test/sql/bgw_scheduler.sql

```diff
@@ -0,0 +1,121 @@
+
+\c single_2 :ROLE_SUPERUSER
+/* 
+ * Note on testing this: we had to add a short pg_sleep in a few places
+ * because WaitForBackgroundWorkerStartup returns when the worker proc has started
+ * not necessarily when it has actually run code or shown up in pg_stat_activity
+ */
+
+CREATE VIEW worker_counts as SELECT count(*) filter (WHERE application_name = 'Timescale BGW Cluster Launcher') as launcher,
+count(*) filter (WHERE application_name = 'Timescale BGW Scheduler Entrypoint' AND datname = 'single') as single_scheduler,
+count(*) filter (WHERE application_name = 'Timescale BGW Scheduler Entrypoint' AND datname = 'single_2') as single_2_scheduler
+FROM pg_stat_activity;
+/* 
+ * when we've connected to single_2, we should be able to see the cluster launcher 
+ * and the scheduler for single in pg_stat_activity
```

Comment:
I'm actually not sure why you should see the scheduler for single? Why is the TS extension loaded in `single` but not `single_2`? (Or is this in runner.sh and I just didn't look closely enough)

Reply:
yep, runner.sh created the extension in single, but not in single_2.